tests/ui/borrowck/borrowck-uninit-binding-suggestion.rs

```diff
@@ -0,0 +1,16 @@
+// regression test for #120634
```

Comment:
there's gotta be a test for this error already somewhere, right? could you try finding it and extending it instead of creating a new one?